src/index.native.tsx

```diff
@@ -138,64 +153,54 @@ const styles = StyleSheet.create({
   },
 });
 
-const ScreenStackHeaderBackButtonImage = (props) => (
+export const ScreenStackHeaderBackButtonImage = (
+  props: ImageProps
+): JSX.Element => (
   <ScreensNativeModules.NativeScreenStackHeaderSubview
     type="back"
     style={styles.headerSubview}>
     <Image resizeMode="center" fadeDuration={0} {...props} />
   </ScreensNativeModules.NativeScreenStackHeaderSubview>
 );
 
-const ScreenStackHeaderRightView = (props) => (
+export const ScreenStackHeaderRightView = (
+  props: PropsWithChildren<ViewProps>
+): JSX.Element => (
   <ScreensNativeModules.NativeScreenStackHeaderSubview
     {...props}
     type="right"
     style={styles.headerSubview}
   />
 );
 
-const ScreenStackHeaderLeftView = (props) => (
+export const ScreenStackHeaderLeftView = (
+  props: PropsWithChildren<ViewProps>
+): JSX.Element => (
   <ScreensNativeModules.NativeScreenStackHeaderSubview
     {...props}
     type="left"
     style={styles.headerSubview}
   />
 );
 
-const ScreenStackHeaderCenterView = (props) => (
+export const ScreenStackHeaderCenterView = (
+  props: PropsWithChildren<ViewProps>
+): JSX.Element => (
   <ScreensNativeModules.NativeScreenStackHeaderSubview
     {...props}
     type="center"
     style={styles.headerSubview}
   />
 );
 
-module.exports = {
-  ScreenContainer,
-  Screen,
-  get NativeScreen() {
-    return ScreensNativeModules.NativeScreen;
-  },
+export const NativeScreen = ScreensNativeModules.NativeScreen;
```

Comment:
I had to remove `module.exports` because `babel` couldn't properly parse having `module.exports` and `export const ...` in one file for TS file. I believe these `get` methods were not necessary since `ScreensNativeModules` uses `get`, but I might be wrong.